hera_mc/mc_session.py

```diff
@@ -252,6 +252,56 @@ def get_server_status(self, subsystem, starttime, stoptime=None, hostname=None):
 
         return status_list
 
+    def add_subsystem_error(self, time, subsystem, severity, log):
+        """
+        Add a new subsystem subsystem_error to the M&C database.
+
+        Parameters:
+        ------------
+        subsystem: string
+            name of subsystem.
+        time: astropy time object
+            time of this error report
+        log: string
+            error message or log file name (TBD)
+        """
+        from .subsystem_error import SubsystemError
+
+        db_time = self.get_current_db_time()
+
+        self.add(SubsystemError.create(db_time, time, subsystem, severity, log))
+
+    def get_subsystem_error(self, starttime, stoptime=None, subsystem=None):
+        """
+        Get subsystem server_status record(s) from the M&C database.
+
+        Parameters:
+        ------------
+        subsystem: string
+            name of subsystem. Must be one of ['rtp', 'lib']
+
+        starttime: astropy time object
+            time to look for records after
+
+        stoptime: astropy time object
+            last time to get records for. If none, only the first record after
+            starttime will be returned.
+
+        subsystem: string
+            subsystem to get records for. If none, all subsystems will be included.
+
+        Returns:
+        --------
+        list of SubsystemError objects
+        """
+        from .subsystem_error import SubsystemError
+
+        error_list = self._time_filter(SubsystemError, 'time', starttime,
```

Comment:
Style comment: could just return this value instead of saving it in a variable that is immediately returned.